updater/spec/bin_run_spec.rb

```diff
@@ -5,20 +5,29 @@
 
 RSpec.describe "bin/run" do
   describe "fetch_files" do
-    before do
-      ENV["DEPENDABOT_JOB_ID"] = "1"
-      ENV["DEPENDABOT_JOB_TOKEN"] = "token"
-      ENV["DEPENDABOT_JOB_PATH"] =
-        "spec/fixtures/jobs/job_with_credentials.json"
-      ENV["DEPENDABOT_OUTPUT_PATH"] = File.join(Dir.mktmpdir, "output.json")
-      ENV["DEPENDABOT_API_URL"] = "http://example.com"
-    end
+    around do |example|
+      Dir.mktmpdir do |tempdir|
+        output_path = File.join(tempdir, "output.json")
+        job_path = File.join(tempdir, "job.json")
+
+        job_info = JSON.parse(File.read("spec/fixtures/jobs/job_with_credentials.json"))
+        job_info["job"]["credentials"][0]["password"] = test_access_token
+
+        File.write(job_path, JSON.dump(job_info))
+
+        ENV["DEPENDABOT_JOB_ID"] = "1"
+        ENV["DEPENDABOT_JOB_TOKEN"] = "token"
+        ENV["DEPENDABOT_JOB_PATH"] = job_path
+        ENV["DEPENDABOT_OUTPUT_PATH"] = output_path
+        ENV["DEPENDABOT_API_URL"] = "http://example.com"
 
-    after do
-      ENV["DEPENDABOT_JOB_ID"] = nil
-      ENV["DEPENDABOT_JOB_TOKEN"] = nil
-      ENV["DEPENDABOT_JOB_PATH"] = nil
-      ENV["DEPENDABOT_API_URL"] = nil
+        example.run
+      ensure
+        ENV["DEPENDABOT_JOB_ID"] = nil
+        ENV["DEPENDABOT_JOB_TOKEN"] = nil
+        ENV["DEPENDABOT_JOB_PATH"] = nil
```

Comment:
I see `ENV["DEPENDABOT_OUTPUT_PATH"]` wasn't cleared out _before_ this change. Should it be?

Reply:
I guess, I don't see how it should be special, let me add that.

Reply:
Done!